Configuration/TCA/Overrides/pages.php

```diff
@@ -0,0 +1,66 @@
+<?php
+defined('TYPO3_MODE') || die('Access denied.');
+
+call_user_func(
```

Comment:
This call is not needed here. Please, make it plain addition as usual if you want this to be merged.